media_driver/agnostic/common/codec/hal/codechal_encode_avc_base.cpp

```diff
@@ -1763,7 +1763,7 @@ MOS_STATUS CodecHalAvcEncode_PackSliceHeader(
     // zero byte shall exist when the byte stream NAL unit syntax structure contains the first
     // NAL unit of an access unit in decoding order, as specified by subclause 7.4.1.2.3.
     // VDEnc Slice header packing handled by PAK does not need the 0 byte inserted
```

Comment:
it means you will remove this sentence , right

Reply:
ok ,i will remove this line